spec/requests/properties_spec.rb

```diff
@@ -187,123 +187,194 @@
+                "streetSuffix" => ""
+              },
+              "hierarchyType" => {
+                "levelCode" => "1",
+                "subTypeCode" => "DWE",
+                "subTypeDescription" => "Dwelling"
+              }
+            },
+            "alerts" => {
+              "locationAlert" => [
                 {
-                  "tenancyAgreementReference": "011111/01"
+                  "type" => "DIS",
+                  "comments" => "Property Under Disrepair",
+                  "startDate" => "2011-02-16",
+                  "endDate" => nil
                 }
-              ]
-            },
-          status: 200
-        )
-
-        stub_alerts_residents_request(
-          tag_ref: "011111/01",
-          response_body:
-            {
-              "contacts": [
+              ],
+              "personAlert" => [
                 {
-                  "tenancyAgreementReference": "011111/01",
-                  "alerts": [
-                    {
-                      "startDate": "2018-04-13",
-                      "endDate": nil,
-                      "alertCode": "CV",
-                      "description": "No Lone Visits"
-                    },
-                    {
-                      "startDate": "2018-04-13",
-                      "endDate": nil,
-                      "alertCode": "VA",
-                      "description": "Verbal Abuse or Threat of"
-                    }
-                  ]
+                  "type" => "CV",
+                  "comments" => "No Lone Visits",
+                  "startDate" => "2018-04-13",
+                  "endDate" => nil
+                },
+                {
+                  "type" => "VA",
+                  "comments" => "Verbal Abuse or Threat of",
+                  "startDate" => "2018-04-13",
+                  "endDate" => nil
                 }
               ]
             },
-          status: 200
-        )
-
-        get("/api/v2/properties/100023022310", headers: headers)
+            "tenure" => {
+              "typeCode" => "SEC",
+              "typeDescription" => "Secure"
+            }
+          )
+        end
       end
 
-      it "returns a JSON representation from a single property" do
-        parsed_response = JSON.parse(response.body)
-
-        expect(parsed_response).to eq(
-          "property" => {
-            "propertyReference" => "100023022310",
-            "address" => {
-              "shortAddress" => "1 Example Road",
-              "postalCode" => "A1 1AA",
-              "addressLine" => "1 Example Road",
-              "streetSuffix" => ""
-            },
-            "hierarchyType" => {
-              "levelCode" => "1",
-              "subTypeCode" => "DWE",
-              "subTypeDescription" => "Dwelling"
-            }
-          },
-          "alerts" => {
-            "locationAlert" => [
+      context "with no tenancy information details" do
+        before do
+          stub_property_request(
```

Comment:
You could probably put this in the block above if it's the same as the above `stub_property_request` https://github.com/LBHackney-IT/repairs-api/pull/21/files#diff-2fe1122181c84f9dbd72fcc3d2629c12530a2a15848dcc3ffc788460f2121f6aR192 to avoid repeating yourself

Reply:
Higher level context block I mean